tarantoolctl

```diff
@@ -0,0 +1,1353 @@
+#!/usr/bin/env tarantool
```

Comment:
> tarantool will be build as a static build and in the future the tarantoolctl utility will be removed

The test-run change is completely unrelated to how tarantool is linked with libraries. It would confuse an external reader, which doesn't know that tarantoolctl deprecation/removal is in the same track as change of the linking type.

It is even more related to the second commit, which calls some layout changes 'static build' not only in the commit message, but also in the option name.

Reply:
Yes, thanks! Fixed it.
Indeed, many people do not care how the build tarantool and this is not the root cause of our changes.